client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx

```diff
@@ -44,7 +44,6 @@ export default function SiteTopHeaderButtons() {
 				<span className="sites-overview__issue-license-button-caption">
```

Comment:
`span.sites-overview__issue-license-button-caption` isn't used anywhere (you can try and remove it in the browser as well), but I assume it had a function in context of being next to the icon.
Should we perhaps remove it as well, if we remove the icon?

Reply:
`span` wrapping element removed. It wasn't there before. 2730ef2d8f27bd73581aae9f58e7398c64e89539